src/ontology/templates/assays.tsv

```diff
@@ -656,6 +656,9 @@ OBI:0003599	single cell T cell receptor repertoire sequencing assay	single cell
 OBI:0003601	quartz crystal microbalance assay	quartz crystal microbalance assay		QCM	An assay that measures the change in frequency of a quartz crystal resonator resulting from the addition or removal of a small mass of a ligand specifically binding at the surface of the resonator.	http://www.ncbi.nlm.nih.gov/pubmed/23504432|https://en.wikipedia.org/wiki/Quartz_crystal_microbalance		Randi Vita|Bjoern Peters|Sebastian Duesing					subclass		assay														
 OBI:0003604	multiplexed error-robust fluorescence in situ hybridization assay			MERFISH	A fluorescence in-situ hybridization assay that uses combinatorial FISH labeling with error resistant encoding schemes for highly multiplexed single-molecule imaging.	https://en.wikipedia.org/wiki/Fluorescence_in_situ_hybridization#MERFISH		Sebastian Duesing			Brain Image Library	https://github.com/obi-ontology/obi/issues/1724	subclass		fluorescence in-situ hybridization assay														
 OBI:0003605	single molecule fluorescence in situ hybridization			smFISH	A fluorescence in-situ hybridization assay that detects single RNA molecules in fixed cells, allowing for both the quantification and localization at the single-cell level and single-molecule resolution.	https://doi.org/10.21769/BioProtoc.3070		Sebastian Duesing			Brain Image Library	https://github.com/obi-ontology/obi/issues/1725	subclass		fluorescence in-situ hybridization assay														
+OBI:0003659	in vitro CRISPR screening assay	in vitro CRISPR screening assay			An assay in which cells expressing a Cas system are infected with a library of guide RNAs with the intent of causing a perturbation at the target of the guide RNAs.			Jennifer Jou|Bjoern Peters			Jennifer Jou	https://github.com/obi-ontology/obi/issues/1784	subclass		assay	cas mediated mutagenesis	assay	cell											
+OBI:0003660	in vitro CRISPR screen using single RNA-Seq	in vitro CRISPR screen using single RNA-Seq			An in vitro CRISPR screening assay that uses single cell RNA-Seq to detect induced changes in gene expression			Jennifer Jou|Bjoern Peters			Jennifer Jou	https://github.com/obi-ontology/obi/issues/1784	subclass		in vitro CRISPR screening assay		single-cell RNA sequencing assay												
```

Comment:
I noticed that the term name says "single RNA-seq"; I'm not sure if that's accidentally omitting the word "cell" (as in "single cell RNA-seq")?

Also, if it's OK, I'd like to share this pull request with some of our labs working on CRISPR screens to see if they are happy with the terminology.

Reply:
@jenjou Good catch—I believe that change was made while Bjoern was editing the template during the OBI call a couple weeks ago. I'll add a commit to change it to "single-cell RNA-Seq". Yes, please feel free to share this PR with the labs.